pallets/gear/src/manager/journal.rs

```diff
@@ -472,16 +472,13 @@ where
             unreachable!("Threshold for reservation invalidated")
```

Comment:
without validation on zero duration you can run into this unreachable - need test also